tests/cli_terraform_test.go

```
@@ -0,0 +1,175 @@
+func runCLITerraformCleanComponent(t *testing.T, binaryPath, environment string) {
+	cmd := exec.Command(binaryPath, "terraform", "clean", "station", "-s", environment, "--force")
+	var stdout, stderr bytes.Buffer
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
+	err := cmd.Run()
+	t.Logf("Clean command output:\n%s", stdout.String())
+	if err != nil {
+		t.Fatalf("Failed to run terraform clean: %v", stderr.String())
+	}
+}
+func runCLITerraformClean(t *testing.T, binaryPath string) {
```

golangci-lint (1.62.2)

func `runCLITerraformClean` is unused

(unused)